foundations/javascript_basics/problem_solving.md

```diff
@@ -71,7 +71,7 @@ In short, break the big problem down and solve each of the smaller problems unti
 
 ### Solving Fizz Buzz
 
-To demonstrate this workflow in action, let's solve [a common programming exercise: Fizz Buzz, explained in this wiki article](https://en.wikipedia.org/wiki/Fizz_buzz).
+let's solve [Fizz Buzz](https://en.wikipedia.org/wiki/Fizz_buzz)
```

Comment:
```suggestion
To demonstrate this workflow in action, let's solve [Fizz Buzz](https://en.wikipedia.org/wiki/Fizz_buzz)
```
The spreadsheet original text only starts from the "let's solve..." part, so be careful to only replace the original, not accidentally the bit before :P

Reply:
Ah. I did it again, my bad >.<